language/move-core/types/src/account_address.rs

```diff
@@ -17,11 +17,21 @@ impl AccountAddress {
     }
 
     /// The number of bytes in an address.
-    pub const LENGTH: usize = 16;
+    /// Default to 16 bytes, can be set to 20 bytes with address20 feature.
+    pub const LENGTH: usize = if cfg!(feature = "address20") { 20 } else { 16 };
```

Comment:
Everything in this PR looks good, not sure if @wrwg or @bmwill have any final thoughts on this direction?